examples/directory job/clusterrunner.yaml

```diff
@@ -1,4 +1,6 @@
 SampleJob:
+    max_executors: 2
+    max_executors_per_slave: 5
```

Comment:
Just adding these here for basic reference purposes. This doesn't really do anything. :smile_cat:.